### PR TITLE
AO3-4998 Can't clarify ambiguous pseud on chapters

### PR DESCRIPTION
**Pull request by sarken**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4998

## Purpose

Makes it so you can clarify the identity of a chapter co-author with an ambiguous pseud. Bonus: Fixes the association between the label and the select menu on the ambiguous pseud page.

## Testing

See issue.

### CODE REVIEW

File: app/views/pseuds/_coauthor_form.html.erb [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 <% unless creation.invalid_pseuds.blank? %> 
-  <h4 class="heading"><%= ts('These pseuds are invalid:') %> </h4>
+  <h4 class="heading"><%= ts("These pseuds are invalid:") %></h4>
   <ul>
   <% for name in creation.invalid_pseuds %>
     <li><%= name %></li>
   <% end %>
   </ul>
-  <p><%= form.label :pseud, ts('Try again:') %></p>
-  <p title="try again">
```
Comment (sarken) [inline app/views/pseuds/_coauthor_form.html.erb]: :sob:

File: features/works/chapter_edit.feature [hunk 2]
```diff
@@ -435,3 +436,22 @@ Feature: Edit chapters
       And I press "Post Without Preview"
     Then I should see "Publication date can't be in the future."
     When I jump in our Delorean and return to the present
+
+
+  Scenario: You should be able to add a chapter with a co-creator who has an
```
Comment (redsummernight) [inline features/works/chapter_edit.feature]: There are two sets of pending tests for AO3-4998 in this file, they should be enabled or removed.

**Comment by sarken** [comment]
This had merge conflicts, but for some reason, they weren't showing up here. :| I've resolved those and uncommented the tests @redsummernight pointed out.

**Comment by sarken** [comment]
This won't be necessary once we make our big co-creator changes.

**Pull request closed without merging**